src/cal.py

```diff
@@ -14,3 +14,29 @@
 # docs for the calendar module closely.
 
 import sys
+import calendar
+from datetime import date
+
+def get_month():
+    try:
+        arg_month = int(sys.argv[1])
+    except (IndexError, ValueError):
```

Comment:
It looks like you're doing something clever with error handling here -- very cool!